### ISSUE TITLE
Add support for Central Package Management 

### ISSUE THREAD

**Issue by stevenvolckaert** [CZEMacLeod/MSBuild.SDK.SystemWeb#46]
The release of NuGet 6.2.0, released with Visual Studio 2022 17.2, makes it possible to manage the versions of package references centrally in a single (or multiple, if one would wish) `Directory.Packages.props` file.

See [Central Package Management (CPM)](https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management) on Microsoft Docs.

I've been migrating some of our projects to use CPM. Today I tried to migrate our first `Sdk="MSBuild.SDK.SystemWeb/4.0.82"` project, but I'm getting a NU1008 error:

> _Projects that use central package version management should not define the version on the PackageReference items but on the PackageVersion items: Microsoft.CodeDom.Providers.DotNetCompilerPlatform;Microsoft.Net.Compilers.Toolset._

I presume MSBuild.SDK.SystemWeb needs an update to support this scenario?

Basically, when a `Directory.Packages.props` file is defined / used, the `Version` attribute on packages `Microsoft.CodeDom.Providers.DotNetCompilerPlatform` and `Microsoft.Net.Compilers.Toolset` should either be removed or be updated to `VersionOverride`.

Would this be a possibility in the near future?

Thank you!

**Comment by CZEMacLeod** [comment]
The SDK already supports the [`Microsoft.Build.CentralPackageVersions` SDK/package](https://github.com/microsoft/MSBuildSdks/tree/main/src/CentralPackageVersions) mechanism for doing this. It detects and omits the version numbers for package references based on the property `UsingMicrosoftCentralPackageVersionsSdk`.
If you add this (set to true) it should work as expected for the NuGet intrinsic system too.
The SDK could be updated to check for the `ManagePackageVersionsCentrally` property too, and I will look at doing that in a future update.

```xml
<Project>
  <PropertyGroup>
    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
    <UsingMicrosoftCentralPackageVersionsSdk>true</UsingMicrosoftCentralPackageVersionsSdk>
  </PropertyGroup>
  <ItemGroup>
    <PackageVersion Include="Microsoft.Net.Compilers.Toolset" Version="4.3.0" />
    <PackageVersion Include="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" Version="3.6.0" />
  </ItemGroup>
</Project>
```

**Comment by stevenvolckaert** [comment]
I confirm this works, thank you for your support @CZEMacLeod!
